### PR TITLE
operator: Add "resources.requests" for CPU and Mem

### PR DESCRIPTION
**Pull request by qinqon**
**Is this a BUG FIX or a FEATURE ?**:
Following best practices for pods configuration and the status quo at
handler. This change ad the resources requests for CPU and memory
following the output of `crictl stats` on the operator container.

> Uncomment only one, leave it on its own line:
>
/kind bug
> /kind enhancement

**What this PR does / why we need it**:

**Special notes for your reviewer**:
The `crictl stats` output peak is

Idle
```bash
CONTAINER           CPU %               MEM                 DISK                INODES
e96d731fdb8e4       0.03                25.02MB             6B                  1
```

Install
```bash
CONTAINER           CPU %               MEM                 DISK                INODES
e96d731fdb8e4       4.00                 25.02MB             6B                  1
```

Delete
```bash
CONTAINER           CPU %               MEM                 DISK                INODES
e96d731fdb8e4       0.06                 25.02MB             6B                  1
```

Tested locally adding also `limits` and it works fine.

**Release note**:

```release-note
Set resources requests for CPU and Memory
```

### CODE REVIEW

File: deploy/operator/operator.yaml [hunk 1]
```diff
@@ -33,6 +33,10 @@ spec:
           imagePullPolicy: {{ .OperatorPullPolicy }}
           command:
           - manager
+          resources:
+            requests:
+              cpu: "60m"
```
Comment (oshoval) [inline deploy/operator/operator.yaml:38]: we sure need the quotes ?

Comment (qinqon) [reply deploy/operator/operator.yaml:38]: Fixed

**Review by oshoval — APPROVED** [review]
Thanks

**Pull request merged**